src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/AdjudicationHearingResultAwardRepository.kt

```diff
@@ -11,18 +11,18 @@ interface AdjudicationHearingResultAwardRepository : JpaRepository<AdjudicationH
   @Query(value = "SELECT NVL(MAX(SANCTION_SEQ)+1, 1) FROM OFFENDER_OIC_SANCTIONS oos WHERE OFFENDER_BOOK_ID = :offenderBookId", nativeQuery = true)
   fun getNextSanctionSequence(offenderBookId: Long): Int
 
-  fun findFirstOrNullByIncidentParty_adjudicationNumberAndSanctionCodeAndHearingResult_chargeSequence(
+  fun findFirstOrNullByIncidentPartyAdjudicationNumberAndSanctionCodeAndHearingResultChargeSequence(
```

Comment:
not sure why underscores had made their way into these repos, but they aren't required and ktlint doesn't like them either